docs/quickstart.md

```diff
@@ -0,0 +1,23 @@
+# Getting Started
+
+## Logging In
+If you already have a TechPass account, you can use TechPass to sign-in to any of the integrated services on SG TechStack. Simple look for the "***Sign-In with TechPass***" button on the services' sign-in page.
+
+If you have do not have a TechPass account yet, follow the instructions below to sign up for one.
+
+## Signing up for an Account
+
+### Public Officer Users with valid gov.sg email
+If you are a user with a **valid gov.sg email account** and have set up your cloud account, you can start by signing up you TechPass account with your gov.sg email on TechPass portal using your **GSIB device**. 

```

Comment:
- Should be "signing up your".
- Use "non-SE GSIB device" instead.

Reply:
done